src/main/java/uk/gov/hmcts/probate/validator/IhtNetQualifyingValueValidationRule.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.probate.validator;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.probate.exception.BusinessValidationException;
+import uk.gov.hmcts.probate.model.ccd.raw.request.CaseDetails;
+import uk.gov.hmcts.probate.service.BusinessValidationMessageRetriever;
+
+import java.math.BigDecimal;
+import java.util.Locale;
+
+@Component
+@RequiredArgsConstructor
+public class IhtNetQualifyingValueValidationRule implements CaseDetailsValidationRule {
+
+    private final BusinessValidationMessageRetriever businessValidationMessageRetriever;
+
+    private static final String IHT_NQV_GT_ESTATE_GROSS = "ihtNQVLargerThanEstateGross";
+    private static final String IHT_NQV_GT_ESTATE_NET = "ihtNQVLargerThanEstateNet";
+    private static final String IHT_NQV_GT_PROBATE_GROSS = "ihtNQVLargerThanProbateGross";
+    private static final String IHT_NQV_GT_PROBATE_NET = "ihtNQVLargerThanProbateNet";
+
+    @Override
+    public void validate(CaseDetails caseDetails) {
+        BigDecimal nqv = caseDetails.getData().getIhtEstateNetQualifyingValue();
+        BigDecimal estateNet = caseDetails.getData().getIhtEstateNetValue();
+        BigDecimal estateGross = caseDetails.getData().getIhtEstateGrossValue();
+        BigDecimal probateNet = caseDetails.getData().getIhtNetValue();
+        BigDecimal probateGross = caseDetails.getData().getIhtGrossValue();
+
+        if (nqv != null) {
+            if (estateGross != null && nqv.doubleValue() > estateGross.doubleValue()) {
```

Comment:
I wondered about the why of converting to doubleValue , thought compareTo would have done the job

Reply:
I could do that - just seems more obvious to me this way

Reply:
CORRECTED